src/protobuf-net.Test/NullListSupport.cs

```diff
@@ -271,20 +258,6 @@ public void BasicPackedLists_PocoList(bool supportNull, Scenario scenario, Outco
         public void BasicGroupedLists_Schema() => CheckSchema<BasicGroupedLists>(false, @"syntax = ""proto3"";
 package ProtoBuf.Test;
 
-message BasicGroupedLists {
-    repeated int32 Int32List = 1 [packed = false];
-    repeated int32 NullableInt32List = 2;
-    repeated string StringList = 3;
-    repeated group SomePoco PocoList = 4;
-}
-message SomePoco {
-    int32 Value = 1;
-}");
-
-        [Fact]
-        public void BasicGroupedLists_Schema_SupportNull() => CheckSchema<BasicGroupedLists>(true, @"syntax = ""proto3"";
```

Comment:
I have removed tests, because I have added similar ones in `Nullables` folder.
Sorry, did not see these tests existed here. Hope you are fine with that